core/cmd/renderer.go

```diff
@@ -89,11 +90,24 @@ func (rt RendererTable) Render(v interface{}) error {
 		return rt.renderOCRKeys(*typed)
 	case *[]Job:
 		return rt.renderJobsV2(*typed)
+	case *webPresenters.LogResource:
+		return rt.renderLogResource(*typed)
 	default:
 		return fmt.Errorf("unable to render object of type %T: %v", typed, typed)
 	}
 }
 
+func (rt RendererTable) renderLogResource(logResource webPresenters.LogResource) error {
+	table := rt.newTable([]string{"ID", "Level", "SqlEnabled"})
+	table.Append([]string{
+		logResource.ID,
+		logResource.Level,
+		strconv.FormatBool(logResource.SqlEnabled),
+	})
+	render("Logs", table)
```

Comment:
would be nice to have a screen shot of what this table looks like in the PR description

Reply:
added~